### ISSUE TITLE
Timeseries classification

### ISSUE THREAD

**Issue by bgonzalezfractal** [unit8co/darts#1473]
**Is your feature request related to a current problem? Please describe.**
Timeseries classification is not feasible in Darts, IoT has excellent data quality and interesting business cases, we've used Darts many times for regression achieving great results in short time, classification should be a feature in the roadmap since its becoming more important each day. 

**Describe proposed solution**
Being the head of the neural the network, I propose creating a subset of models that can tackle classification, RandomForest and Regresinos with scikit learn could raise a first a opportunity to have time series classification.

**Describe potential alternatives**
As a first approach Darts could take this as a regression problem with a distribution of probability between 0 and 1, and using a function like softmax to adjust to it to a class.

**Additional context**
-

**Comment by tuomijal** [comment]
I agree that there is a specific set of problems that are a combination of time series forecasting and classification. What do you think @madtoinou and @dennisbader, is this something that could fit in the projects scope?

I actually looked into this already and converting LightGBM into classifier is as simple as 
model=lgb.LGBMRegressor(**self.kwargs),
